gym_snake/envs/snakeRewardFuncs.py

```diff
@@ -0,0 +1,26 @@
+"""
+This file is used to store reward functions which we define that follow the basic pattern
+
+inputs:
+    did_consume_fruit: boolean representing whether a snake consumed a fruit in the last move
+    did_collide_wall: boolean representing whether a snake collided with itself in the last move
+    did_collide_body: boolean representing whether a snake collided with a wall in the last move
+
+output: 
+    an integer representing a reward assigned to the snake agent based on the inputs provided
```

Comment:
Let's talk live about this some more. I think that these 3 inputs are pretty constraining. Doesn't provide much flexibility in the kinds of rewards we can define. For example, maybe we want a rewards function that rewards a snake for moving closer to the fruit. Or a reward function that punishes for moving further from wall. Perhaps this can be done by taking the board state as input.

Reply:
I totally agree and am happy to talk about it live. Maybe during our Tuesday huddle we can figure out inputs and then I'll update this PR with them.